### ISSUE TITLE
Merge cells toggling does not work in RTL

### ISSUE THREAD

**Issue by budnix** [handsontable/handsontable#9362]
### Description

Once the cells are merged using the Ctrl+M keyboard shortcut the cell won't be toggled to the unmerged state. The issue is happening only when the table is rendered in RTL mode.

LTR:
![Kapture 2022-03-31 at 12 54 00](https://user-images.githubusercontent.com/571316/161039466-db72b3d4-e169-4fae-934b-705d9ff4f73e.gif)

RTL (bug):
![Kapture 2022-03-31 at 12 54 51](https://user-images.githubusercontent.com/571316/161039631-962220fd-4127-4099-bde9-fffcba08d3c1.gif)

### Demo

https://jsfiddle.net/budnix/2zsrp89h/

### Your environment
* Handsontable version: `develop`
* Browser name and version: any
* Operating system: any

**Comment by warpech** [comment]
@budnix is this issue invalidated by https://github.com/handsontable/handsontable/issues/9368?

**Comment by budnix** [comment]
> @budnix is this issue invalidated by #9368?

No, this issue is a separate bug not related to https://github.com/handsontable/handsontable/issues/9368.

**Comment by Meihualu-cc** [comment]
i have a quesiton about mergeCell & redo undo in handsontable, after several operations of merge and unmerge, i can't return to the initial state. it seems the redo&undo statck losts some previous events.

**Comment by AMBudnik** [comment]
Hi @Meihualu-cc This issue is our RTL. Could you please share the steps to replicate the issue with me on email (support@handsontable.com)?

**Comment by Meihualu-cc** [comment]
> Hi @Meihualu-cc This issue is our RTL. Could you please share the steps to replicate the issue with me on email ([support@handsontable.com](mailto:support@handsontable.com))?

I tried the example on the handsontable docs, the function of redo&undo  is normal after merges . maybe there is something wrong with my code. I will check it later.
